scripts/stackOperation.py

```diff
@@ -0,0 +1,63 @@
+def isEmpty(stk):
```

Comment:
```suggestion
def isEmpty(stk):
    return not stk
```

Reply:
To clarify - I was suggesting that this is a simpler way to write this function so that instead of needing the whole `if ... else` statement, you can do it with just this one return line, so you'll need to replace the whole function with my suggested code